games-fps/urbanterror/urbanterror-4.3.2_p20171016.ebuild

```diff
@@ -0,0 +1,152 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+inherit flag-o-matic gnome2-utils
+
+DESCRIPTION="Hollywood tactical shooter based on the ioquake3 engine"
+HOMEPAGE="http://urbanterror.info https://github.com/mickael9/ioq3"
+
+if [[ ${PV} == "9999" ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/mickael9/ioq3.git"
+	EGIT_BRANCH="urt"
+else
+	COMMIT_ID="3e555ab1405c38556cbfcdd42eac23a4af49e633"
+	SRC_URI="https://github.com/mickael9/ioq3/archive/${COMMIT_ID}.tar.gz -> ${P}.tar.gz"
+	S="${WORKDIR}/ioq3-${COMMIT_ID}"
+	KEYWORDS="~amd64 ~x86"
+
+fi
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="+altgamma +client +curl debug mumble openal +opus server voip vorbis"
+REQUIRED_USE=" || ( client server )"
+
+PATCHES=( "${FILESDIR}"/${PN}-4.3-respect_CFLAGS.patch )
+
+RDEPEND="
+	client? (
+		media-libs/libsdl2[X,sound,joystick,opengl,video]
+		mumble? ( media-sound/mumble )
+		openal? ( media-libs/openal )
+		opus? ( media-libs/opusfile )
+		vorbis? (
+			media-libs/libogg
+			media-libs/libvorbis
+		)
+	)
+	curl? ( net-misc/curl )
+	~games-fps/urbanterror-data-4.3.2
+	sys-libs/zlib[minizip]
+	virtual/jpeg:0
+"
+
+DEPEND="${RDEPEND}"
+
+pkg_pretend() {
+	if use client; then
+		if ! use openal && ! use opus && ! use vorbis; then
+			ewarn
+			ewarn "No sound implementation selected. Enable 'openal', 'opus' or 'vorbis' USE flag to get sound!"
+		fi
+	fi
+}
+
+src_compile() {
+	buildit() { use $1 && echo 1 || echo 0 ; }
+	nobuildit() { use $1 && echo 0 || echo 1 ; }
```

Comment:
Don't use `buildit` and `nobuildit` - see https://devmanual.gentoo.org/ebuild-writing/functions/src_configure/configuring/index.html For example, consider `usex` instead

Reply:
fixed.